plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -120,9 +120,11 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
 
   render() {
     const { nodeHealthResponse, filteredNodes, masterRegion } = this.state;
+    const { networks = [] } = this.props;
```

Comment:
Is the default necessary because sometimes data.networks is not an array? Or just being doubly safe? 😏

Reply:
I guess double safe, since I'm `catchError'ing`  404 and returning an empty array there.